.gitignore

```diff
@@ -35,8 +35,3 @@ cmake-build-*
 *.out
 *.app
 
-# build folders
```

Comment:
Why is this deleted and why is the dockerignore deleted?

Reply:
they are not used anymore and somehow we still have them. This used to be the older build structure so there is no point of having them anymore